EIPS/eip-draft-returndata.md

```diff
@@ -0,0 +1,31 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Embedding transaction return data in receipts
+    Author: Nick Johnson <nick@ethereum.org>
+    Type: Standard Track
+    Category Core
+    Status: Draft
+    Created: 2017-06-30
+    Requires: 140
+    Replaces: 98
+
+
+## Abstract
+This EIP replaces the intermediate state root field of the receipt with a status code indicating if the top-level call succeeded or failed.
+
+## Motivation
+With the introduction of the REVERT opcode in EIP140, it is no longer possible for users to assume that a transaction failed iff it consumed all gas. As a result, there is no clear mechanism for callers to determine whether a transaction succeeded and the state changes contained in it were applied.
+
+Full nodes can provide RPCs to get a transaction return status and value by replaying the transaction, but fast nodes can only do this for nodes after their pivot point, and light nodes cannot do this at all, making a non-consensus solution impractical.
+
+Instead, we propose to replace the intermediate state root, already obsoleted by EIP98, with the return status (1 for success, 0 for failure). This both allows callers to determine success status, and remedies the previous omission of return data from the receipt.
```

Comment:
As far as I understand (from https://github.com/ethereum/EIPs/pull/658#issuecomment-326594093), this particular EIP will be given number 98. If so, it will be confusing if this EIP references EIP 98 as if it was a different one.

Reply:
Should read fine IMO if `, already obsoleted by EIP98,` is removed.

IMO the following ` (1 for success, 0 for failure)` can be dropped, too, since this detail is elaborated on in section Specification, just a few lines below.

Reply:
Remind me, what's RPCs?

Reply:
Regarding the mention of `EIP98`, I would add a link to the relevant issue or pull-request. I believe the `Motivation` section can talk about the outer world rather freely.